<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Заполнение областей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="solve">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Решить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>470</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="colorCutterComboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Красный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Зеленый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Синий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Черный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Желтый</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Цвет:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeCutter">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Замкнуть отсекатель</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeCutting">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Замкнуть отсекаемое</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Отсекатель</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cutterComboBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Прямоугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Многоугольник</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cuttingComboBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Отрезок</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Многоугольник</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Цвет:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Отсекаемое</string>
    </property>
   </widget>
   <widget class="QComboBox" name="colorCuttingComboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Красный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Зеленый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Синий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Черный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Желтый</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
